unicodetools/src/main/java/org/unicode/tools/emoji/GenerateEmojiData.java

```diff
@@ -781,7 +781,7 @@ void show(
                     out.write("\n\n");
                 } else {
                     out.write("# All omitted code points have " + title + "=No\n");
-                    out.write("# @missing: 0000..10FFFF  ; " + title + " ; No\n");
+                    //out.write("# @missing: 0000..10FFFF  ; " + title + " ; No\n");
```

Comment:
```suggestion
                    // out.write("# @missing: 0000..10FFFF  ; " + title + " ; No\n");
```
... to make the style checker happy: https://github.com/unicode-org/unicodetools/runs/7637978590?check_suite_focus=true